app/components/editItemForm.tsx

```diff
@@ -0,0 +1,119 @@
+"use client";
+import React from "react";
+import Button from "@mui/material/Button";
+import Container from "@mui/material/Container";
+import Grid from "@mui/material/Grid";
+import MenuItem from "@mui/material/MenuItem";
+import Select from "@mui/material/Select";
+import Paper from "@mui/material/Paper";
+import Box from "@mui/material/Box";
+import { ShopItem, ShopItems } from "@/lib/features/items/itemsSlice";
+import { useAppDispatch, useAppSelector } from "@/lib/hooks";
+import { useForm, SubmitHandler } from "react-hook-form";
+import { TextField } from "@mui/material";
+import { updateItem, deleteItem } from "@/utils/client";
+
+interface Props {
+  item: ShopItem;
+  itemKey: number;
+}
+
+export default function EditItemForm(props: Props) {
```

Comment:
```suggestion
const EditItemForm: FC<Props> = ({ item, itemKey }) => {
```
should use depressurization